crates/bevy_ui/src/entity.rs

```diff
@@ -32,6 +35,8 @@ pub struct NodeBundle {
     pub global_transform: GlobalTransform,
     /// Describes the visibility properties of the node
     pub visibility: Visibility,
+    /// Algorithmically-computed indication of whether an entity is visible and should be extracted for rendering
+    pub computed_visibility: ComputedVisibility,
```

Comment:
This PR shows a lot of duplication in Bundle structs. It might be worth considering a macro or some other way to compose bundles.

Reply:
Yeah; we should consider using nested bundles here in the future.

Reply:
Yeah finding a way to compose bundles is worth considering. We already have "bundle nesting" but I don't like the ux of that on the user side (flat lists of components are way easier to work with and understand). Open to exploring things that improve on this, ideally while keeping flat lists (or at least, no arbitrary levels of bundle nesting).

Reply:
It would be nice if there were a way to expand a bundle into another bundle, so you could do something like:
```rust
(
    ..BundleA {},
    ..BundleB {},
)
```